ServerCore/Pages/Shared/_EventNavigationPartial.cshtml

```diff
@@ -399,6 +399,9 @@
         }
 
         function updateToastCount() {
+            var unreadCounter = document.querySelector("#unreadCounter");
+            if (!unreadCounter) return;
```

Comment:
In what cases would this be hit? Should this log an error?

Reply:
It's hit when a player has not yet registered. I don't think we need an error.